include/xsimd/config/xsimd_cpuid.hpp

```diff
@@ -75,6 +76,15 @@ namespace xsimd
                 neon64 = 0;
                 best = neon::version() * neon;
 
+#elif defined(__ARM_FEATURE_SVE) && defined(__ARM_FEATURE_SVE_BITS) && __ARM_FEATURE_SVE_BITS > 0
```

Comment:
this shoudla ctually use the `XSIMD_WITH_XXX` macro definition, but that's already how the remaining part of the code are coded, so let's delay that to another patch.